WTWSMS/events/religious_events.txt

```diff
@@ -3634,11 +3680,11 @@ character_event = {
 				trait = zealous
 			}
 			modifier = {
-				factor = 3
+				factor = 5
```

Comment:
It's an increase of two. Cynics are far more likely to convert as a matter of convenience, and that should be reflected accordingly.